CONTRIBUTING.md

```diff
@@ -93,15 +93,18 @@ Writing good commit logs is important. A commit log should describe what
 changed and why. Follow these guidelines when writing one:
 
 1. The first line should be 50 characters or less and contain a short
-   description of the change prefixed with the name of the changed
-   subsystem (e.g. "net: add localAddress and localPort to Socket").
+   description of the change. All words in the description should be in
+   lowercase with the exception of the ones that refer to code (e.g.
```

Comment:
Added a commit to specify that proper nouns and acronyms should not be lowercased.
If we are forgetting something I think we can always add it later.